### PR TITLE
Add analyses as fourth feature on the main page

### LINKED ISSUE #711: Add analyses as fourth "feature" on the main page
Add fourth icon (with description) between data submission and api documentation.

### PR DESCRIPTION
**Pull request by kammerer**
Fixes #711 

See the [preview image](https://pageshot.net/E5q69brPSYjBRcuw/lvh.me)

Please, let me know if you want to modify the text.

### CODE REVIEW

**Comment by wjurkowski** [comment]
I propose: "Perform genome-wide association study to identify genetic variants underpinning phenotypic trait variability"
Later, once we will have more analysis types e.g. integrating transcriptomics data, we will replace this with something less specific

**Comment by kammerer** [comment]
I updated the text with Wiktor's version.

File: app/assets/stylesheets/partials/_sections.scss [hunk 1]
```diff
@@ -73,13 +73,37 @@
       }
     }
 
-    &.submit {
+    &.submit, &.analyse {
```
Comment (Nuanda) [inline app/assets/stylesheets/partials/_sections.scss]: We repeat the same colour twice here. I understand that this is to keep things consistent. But can we try one more variation - to use the second colour from the BIP logo? Maybe that one will suit nicely?

**Comment by wjurkowski** [comment]
@Nuanda Thanks for reminding this. I will add this as soon as I get it directly from Ben.

**Comment by wjurkowski** [comment]
How to cite us

BIP
Eckes AH, Gubała T, Nowakowski P et al. Introducing the Brassica Information Portal: Towards integrating genotypic and phenotypic Brassica crop data [version 1; referees: awaiting peer review]. F1000Research 2017, 6:465 (doi: 10.12688/f1000research.11301.1)

GWASSER
https://github.com/cyverseuk/GWASSER

File: app/assets/stylesheets/partials/_sections.scss (hunk 1 above)
Comment (wjurkowski) [inline app/assets/stylesheets/partials/_sections.scss]: sounds good

Comment (kammerer) [inline app/assets/stylesheets/partials/_sections.scss]: Have a look [here](https://pageshot.net/3h43O94NfF6vgg4I/lvh.me). For me it's ok. Contrast on the button is a bit low, but we are not going to use this colour a lot.

Comment (wjurkowski) [inline app/assets/stylesheets/partials/_sections.scss]: Looks good to me!

**Comment by kammerer** [comment]
I do not see any info about referencing GWASSER, so I added a simple note (and a link): 

> GWASSER tool authored by Ben J. Ward is employed by BIP to perform data analysis.

**Comment by wjurkowski** [comment]
Could we just have the link under the How to cite section as I proposed above? Once we have description of what is it doing we will add it on 'Perform new GWAS' page

**Comment by kammerer** [comment]
All right, changed.

File: app/assets/stylesheets/partials/_sections.scss (hunk 1 above)
Comment (Nuanda) [inline app/assets/stylesheets/partials/_sections.scss]: Agreed.

**Comment by kammerer** [comment]
Can I merge this branch?

**Comment by Nuanda** [comment]
That depends on whether you are able (and should?) get rid of build errors, first. Could you take a look and see which ones you can fix?

**Comment by kammerer** [comment]
@Nuanda Spec failures are not related to these changes. I managed to fix some. I will merge and then try to look into the ones related to GWAS.

**Pull request merged**